src/internal/connector/graph/shared.go

```diff
@@ -0,0 +1,19 @@
+package graph
+
+// shared.go
+// Contains helper methods that are common across multiple
+// Microsoft Graph Applications.
+
+import (
+	"github.com/alcionai/corso/src/internal/common/ptr"
+	"github.com/microsoftgraph/msgraph-sdk-go/models"
+)
+
+func UnwrapAddress(contact models.Recipientable) string {
```

Comment:
This isn't actually shared across multiple services at this time..  Rather than generating a new file we may not need, can this go into the exchange API instead?

Reply:
We need this for OneDrive and SharePoint